client/session.go

```diff
@@ -404,7 +406,7 @@ func (s *session) Fetch(id string, startInclusive, endExclusive time.Time) (enco
 	return iter, nil
 }
 
-func (s *session) FetchAll(ids []string, startInclusive, endExclusive time.Time) (encoding.SeriesIterators, error) {
+func (s *session) FetchAll(idns []idWithNamespace, startInclusive, endExclusive time.Time) (encoding.SeriesIterators, error) {
```

Comment:
This is a public facing API, we can't make one of the types here be a non-exported type =[

Perhaps just a string for namespace as well and then you can only fetch from a specific namespace.